handlers/lobbies.js

```diff
@@ -51,7 +51,7 @@ Dota2.Dota2Client.prototype.createTournamentLobby = function(password, tournamen
         "pass_key": password
     };
 
-    if (tournament_id) {
+    if (tournament_id > 0) {
```

Comment:
The previous condition (`tournament_game_id > 0`) didn't make much sense to me. Whether or not the game should be a tournament game should be decided by the `tournament_id` being set, and not `tournament_game_id`, which doesn't really make senese in the context of Bo1s.